aiohttp_csrf/__init__.py

```diff
@@ -0,0 +1,118 @@
+from functools import wraps
+
+from aiohttp import web
+
+
+from .policy import AbstractPolicy
+from .storage import AbstractStorage
+
+
+__version__ = '0.0.1'
+
+REQUEST_STORAGE_KEY = 'aiohttp_csrf_storage'
+DECORATOR_FIELD_NAME = 'aiohttp_csrf_check'
+
+
+async def get_token(request):
+    storage = request.get(REQUEST_STORAGE_KEY)
```

Comment:
checking `if REQUEST_STORAGE_KEY in request`
 would be faster and also it won't handle cases when None is stored in at REQUEST_STORAGE_KEY